bundles/org.eclipse.ui.workbench/.settings/.api_filters

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You shouldn't need this file if you set proper baseline in your IDE (should now be 4.25 release)

Reply:
OK, but after removing this file (and after upgrading the IDE & baseline), I still get this error message on MANIFEST.MF, similarly to what happened to me previously:

    The major version should be incremented in version 3.127.0, since API breakage occurred since version 3.126.100

And the same goes for the new constants in SearchPattern:

    The field org.eclipse.ui.dialogs.SearchPattern.RULE_SUBSTRING_MATCH has been added to a class

Can you please give advice on how to cope with this then? Or maybe this file is actually still necessary?

Reply:
OK, I see. Let's keep it then. The rule is defined in https://wiki.eclipse.org/Evolving_Java-based_APIs_2#Evolving_API_Classes
```
Adding an API field to an API class that is extended by Clients is dangerous in two respects:

    It may break API contract compatibility similar to [Evolving_Java-based_APIs#Example_4_-_Adding_an_API_method](https://wiki.eclipse.org/Evolving_Java-based_APIs#Example_4_-_Adding_an_API_method) in case of name clashes.
    It may cause linkage errors in case an instance (respectively static) field hides a static (respectively instance) field, see [JLS8 13.4.8](https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.4.8). This can not happen if all field declarations follow the usual naming conventions from [JLS8 6.1](https://docs.oracle.com/javase/specs/jls/se8/html/jls-6.html#jls-6.1) and classes only have API fields that are either
        "static final" (and hence have a name that doesn't contain any lowercase letters), or
        non-static and non-final (and hence have a name that contains a least one lowercase letter)

Apart from the binary compatibility issues, it is generally good software engineering practice that API classes should not expose any non-constant fields.
```
and I believe we're in the case where all API (non-private) fields in SearchPattern are `static final`

Reply:
OK. Note that by coincidence, this file emerged in the meantime in master, so I have put the corresponding lines to it after doing Git rebase.

Reply:
Considering this done.